tests/sla_tests.js

```diff
@@ -40,8 +40,14 @@ var config = 'tests/testConfigs/singleEndpoint.json';
 var mockDeviceStates = [];
 // Device state object
 var ds = {};
+// Whether or not moh has started on the bridge
+var mohStarted = false;
```

Comment:
I am confused why you need 2 variables to denote the same thing.

If mohStarted = false, does that not imply that mohStopped = true?

I do not like the notion of maintaining lots of state variables - it always becomes cumbersome and difficult to unwind when it is time to revisit the code.